lib/content-services/src/lib/content-node-share/content-node-share.dialog.scss

```diff
@@ -66,4 +66,19 @@
     .mat-form-field-flex {
         align-items: center;
     }
+
+    @media screen and (max-width: 380px) {
+        .mat-dialog-container {
+            padding: 0 15px;
```

Comment:
just wanted to confirm if that is intended. Your css changes will affect the whole application in 380px width because your ViewEncapsulation is set to None. If you want to limit your changes to the scope of content-node-share-dialog component then you need a different selector here or remove the ViewEncapsulation.None from the component so that the default Angular encapsulation takes place which is ViewEncapsulation.Emulated https://angular.io/guide/view-encapsulation . I am not suggesting you change that, but I am suggesting make sure what you are doing is actually what you need, and not accidentally affect the whole application

Reply:
@arditdomi from what I see, this style is scoped to the parent `.adf-share-link-dialog`

Reply:
sounds good. Thanks

Reply:
hi @arditdomi are your queries cleared now, can we close this PR?

Reply:
Yes go ahead, thanks!

Reply:
sure, thanks for confirmation !!